OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -307,6 +307,29 @@ public GLProfile[] SupportedGLProfiles
 				{
 					SDL.SDL_SetWindowFullscreen(Window, (uint)SDL.SDL_WindowFlags.SDL_WINDOW_FULLSCREEN_DESKTOP);
 					SDL.SDL_SetHint(SDL.SDL_HINT_VIDEO_MINIMIZE_ON_FOCUS_LOSS, "0");
+
+					if (Platform.CurrentPlatform == PlatformType.OSX)
+					{
+						// Activating SDL_WINDOW_FULLSCREEN_DESKTOP on a display with a notch will automatically
+						// reduce the window height and align the top-left of the window to the safe area.
+						//
+						// SDL (as of version 2.26) does not contain an API to query the safeAreaInsets before
+						// the window is created. We work around this by checking the window height after going
+						// fullscreen, and recalculating our sizes to match the new window geometry.
+						//
+						// This workaround will become redundant once window resizing is implemented.
+						SDL.SDL_GetWindowSize(Window, out var width, out var height);
+						if (height != windowSize.Height)
+						{
+							windowSize = new Size(width, height);
+
+							SDL.SDL_GL_GetDrawableSize(Window, out width, out height);
+							surfaceSize = new Size(width, height);
+							windowScale = width * 1f / windowSize.Width;
+
+							Console.WriteLine($"Using new resolution: {windowSize.Width}x{windowSize.Height}");
```

Comment:
This feels a bit weird considering all of the above Console reports will now be wrong. Perhaps we could move the window scale report to be bellow this? It seems it was reported too early anyways

Reply:
Ok, done.